mkdocs/utils.py

```diff
@@ -35,6 +36,39 @@
     text_type = unicode         # noqa
 
 
+def yaml_load(source, loader=yaml.Loader):
+    """
+    Custom yaml loader.
+
+    Load all strings as unicode.
+    http://stackoverflow.com/a/2967461/3609487
+    """
+
+    class Loader(loader):
+
+        """Custom Loader."""
+
+        pass
```

Comment:
I don't want to monkey patch the global loader.  I don't think that is ever a good approach.  Instead, I inherit from the global loader and use my own leaving the global intact in its default state.  This I feel is a better approach.  I can, instead of removing the comment, expound on this more in the class comment.  I apologize as sometimes what is obvious in my head isn't obvious outside of it :).